openmetadata-service/src/main/java/org/openmetadata/service/util/MicrometerBundleSingleton.java

```diff
@@ -15,33 +15,54 @@
 
 import static io.github.maksymdolgykh.dropwizard.micrometer.MicrometerBundle.prometheusRegistry;
 
+import com.azure.core.annotation.Get;
 import io.github.maksymdolgykh.dropwizard.micrometer.MicrometerBundle;
 import io.micrometer.core.instrument.Timer;
 import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.prometheus.client.Histogram;
+import lombok.Getter;
 import org.openmetadata.service.OpenMetadataApplicationConfig;
 
+
 public class MicrometerBundleSingleton {
-  private static final MicrometerBundle instance = new MicrometerBundle();
+  @Getter private static final MicrometerBundle instance = new MicrometerBundle();
   // We'll use this registry to add monitoring around Ingestion Pipelines
   public static final PrometheusMeterRegistry prometheusMeterRegistry = prometheusRegistry;
-  private static Timer webAnalyticEvents;
+  @Getter private static Timer requestsLatencyTimer;
+  @Getter private static Timer jdbiLatencyTimer;
 
-  private MicrometerBundleSingleton() {}
 
-  public static MicrometerBundle getInstance() {
-    return instance;
+  private MicrometerBundleSingleton() {
   }
 
-  public static void setWebAnalyticsEvents(OpenMetadataApplicationConfig config) {
-    webAnalyticEvents =
-        Timer.builder("latency_requests")
-            .description("Request latency in seconds.")
+  private static final double[] latencyBuckets = new double[]
+      {.01, .1, 1, 2, 5, 10, 20, 60, 300};
```

Comment:
LMK if you see the need on having these as a config cc @tutte

Reply:
Static is ok, we can remove the 300 as the HTTP should kill every request beyond 60s